src/Files.App/Views/LayoutModes/StandardLayoutMode.cs

```diff
@@ -248,11 +248,13 @@ protected async void RenameTextBox_KeyDown(object sender, KeyRoutedEventArgs e)
 					e.Handled = true;
 					break;
 				case VirtualKey.Up:
-					textBox.SelectionStart = 0;
+					if (textBox.SelectionStart != 0)
```

Comment:
I think it might be cleaner to check if the `shift` key is pressed. Let me know if you have any questions on how to do this.

Reply:
Yeah, I'm now checking for the shift key